staging/src/k8s.io/api/flowcontrol/v1beta3/types.go

```diff
@@ -472,6 +472,30 @@ type LimitedPriorityLevelConfiguration struct {
 
 	// `limitResponse` indicates what to do with requests that can not be executed right now
 	LimitResponse LimitResponse `json:"limitResponse,omitempty" protobuf:"bytes,2,opt,name=limitResponse"`
+
+	// `lendablePercent` prescribes the fraction of the level's NominalCL that
+	// can be borrowed by other priority levels.  This value of this
+	// field must be between 0 and 100, inclusive, and it defaults to 0.
+	// The number of seats that other levels can borrow from this level, known
+	// as this level's LendableConcurrencyLimit (LendableCL), is defined as follows.
+	//
+	// LendableCL(i) = round( NominalCL(i) * lendablePercent(i)/100.0 )
+	//
+	// +optional
+	LendablePercent int32 `json:"lendablePercent" protobuf:"varint,3,opt,name=lendablePercent"`
+
+	// `borrowingLimitPercent`, if present, specifies a limit on how many seats
+	// this priority level can borrow from other priority levels.  The limit
+	// is known as this level's BorrowingConcurrencyLimit (BorrowingCL) and
+	// is a limit on the total number of seats that this level may borrow
+	// at any one time.  When this field is non-nil, it must hold a non-negative
```

Comment:
This is probably a copy of words I wrote, but now that I read them here I see a problem.  The words here say this field holds a number of seats; the truth is that this field holds a ratio.  Perhaps words like the following would be better?

```
// `borrowingLimitPercent`, if present, configures a limit on how many seats
// this priority level can borrow from other priority levels.  The limit
// is known as this level's BorrowingConcurrencyLimit (BorrowingCL) and
// is a limit on the total number of seats that this level may borrow
// at any one time.
// This field holds the ratio of that limit to the level's nominal concurrency limit.
// When this field is non-nil, it must hold a non-negative

Reply:
The KEP change just merged.  Let's not worry about wording details there, just fix it here.